src_py/apiServer/receiver.py

```diff
@@ -128,4 +132,36 @@ def processResult(resData, resList, currentPhase):
         resList[dictIdx][worker] = [result]
     else:
         resList[dictIdx][worker].append(result)
+"""
+
+"""
+while True:
+                print("1) Use the default address (http://127.0.0.1:8095).")
+                print("2) Enter an address manually.")
+                print("\nPlease choose an option:", end = ' ')
```

Comment:
add an exit option

Reply:
This part was abandoned, because it is not posisible to edit the ServerAPI address after Erlang has already loaded the JSON file (which specifies the address also for Erlang).